packages/compass-indexes/src/modules/search-indexes.ts

```diff
@@ -58,6 +58,13 @@ export enum ActionTypes {
   SetSearchIndexes = 'indexes/search-indexes/SetSearchIndexes',
   SearchIndexesSorted = 'indexes/search-indexes/SearchIndexesSorted',
   SetError = 'indexes/search-indexes/SetError',
+
+  // Drop Search Index
+  OpenDropSearchIndexModal = 'indexes/search-indexes/OpenDropSearchIndexModal',
+  CloseDropSearchIndexModal = 'indexes/search-indexes/CloseDropSearchIndexModal',
```

Comment:
Seems like it's the same as `showConfirmation`? Do we need this in the state and all the components with it?

Reply:
I didnt use that because this drop modal requires users to input the name to confirm action and also has additional subtitle and error state, which is not offered by `showConfirmation`. 

Now that I explored the LG Confirmation Modal, they offer to have confirmation input. So i'll switch to that and clean up the code.

Reply:
added in [757d865](https://github.com/mongodb-js/compass/pull/4844/commits/757d865c7e8bf5d3902207a0932ea4acf34045c7)